Signal/src/ViewControllers/DebugUI/DebugUIMessages.m

```diff
@@ -3399,11 +3401,12 @@ + (void)insertAndDeleteNewOutgoingMessages:(NSUInteger)count
         NSString *text = [self randomText];
         OWSDisappearingMessagesConfiguration *configuration =
             [OWSDisappearingMessagesConfiguration fetchObjectWithUniqueID:thread.uniqueId transaction:transaction];
-        TSOutgoingMessage *message =
-            [TSOutgoingMessage outgoingMessageInThread:thread
-                                           messageBody:text
-                                          attachmentId:nil
-                                      expiresInSeconds:(configuration.isEnabled ? configuration.durationSeconds : 0)];
+
+        uint32_t expiresInSeconds = (configuration.isEnabled ? configuration.durationSeconds : 0);
+        TSOutgoingMessage *message = [TSOutgoingMessage outgoingMessageInThread:thread
+                                                                    messageBody:text
+                                                                   attachmentId:nil
+                                                               expiresInSeconds:expiresInSeconds];
```

Comment:
Maybe we should modify the `[TSOutgoingMessage outgoingMessageInThread:...` factory method to take a nullable `OWSDisappearingMessagesConfiguration`, to DRY this up.

No need to do this in this PR, just thinking out loud.